examples/Advanced-Building-your-own-architecture.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>I think that is out of scope for this release.</p><p> </p><p>Why do these Blocks have the postfix <code>Block</code> and the other ones do not? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>As we discussed, I replaced this section by <code>Connects methods</code> listing the methods of the Block class that implement the abstraction to these different combinators.  Regarding your question, I think we chose to add the postfix <code>Block</code> as these classes are applied to a group of <code>Blocks</code>.   </p>